app/assets/javascripts/discourse/app/controllers/user-private-messages.js

```diff
@@ -17,6 +19,7 @@ export default Controller.extend({
   group: null,
   groupFilter: alias("group.name"),
   currentPath: alias("router._router.currentPath"),
```

Comment:
How does this one work previously without having the router service injected? Also should it be `router.currentPath`, or maybe the new one should be `router._router.currentRouteName`?

Reply:
This line is totally broken and I've created a PR to fix it.